ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml

```
@@ -157,10 +157,14 @@ images:
   dockerfile_path: Dockerfile
   from: base-c9s-python-3.9
   to: amd-c9s-python-3.9
-- context_dir: jupyter/minimal/ubi9-python-3.9
+- context_dir: jupyter/amd/tensorflow/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: jupyter-datascience-ubi9-python-3.9
+  to: amd-jupyter-tensorflow-c9s-python-3.9
+- context_dir: jupyter/amd/pytorch/ubi9-python-3.9
```

We need to build minimal,datascience with AMD
however we don't push them to quay.io, they would only be used for building 
pytorch and tensorflow.

```suggestion
- context_dir: jupyter/minimal/ubi9-python-3.9
  dockerfile_path: Dockerfile
  from: amd-c9s-python-3.9
  to: amd-jupyter-minimal-c9s-python-3.9
- context_dir: jupyter/data-science/ubi9-python-3.9
  dockerfile_path: Dockerfile
  from: amd-jupyter-minimal-c9s-python-3.9
  to: amd-jupyter-datascience-c9s-python-3.9
- context_dir: jupyter/amd/tensorflow/ubi9-python-3.9
  dockerfile_path: Dockerfile
  from: amd-jupyter-datascience-c9s-python-3.9
  to: amd-jupyter-tensorflow-c9s-python-3.9
- context_dir: jupyter/amd/pytorch/ubi9-python-3.9
```



Even though we are using c9s base image, we use the structure of ubi9-python3.9 in others
as there is no OS based information.